docs/components/links/README.md

```diff
@@ -278,14 +278,17 @@ This component has compliance with following WebAIM’s accessibility guidelines
 
 ## Don't Use When
 
-- Navigating a user from promotional or campaign content. Instead, use [Call to Action](../cta/)
+- Triggering interface interactions, such as opening an modal or progressing a user through a flow. Instead, use [Buttons](../buttons/)
+- Navigating a user from promotional or campaign content. Instead, use [Buttons](../buttons/) styled to match the campaign
 
 ## The Basics
 
 - Link styles are adapted based on context, such as for links included in the [Breadcrumb](../breadcrumb/), Menus, and Navigation
 - There are 2 basic link styles: default and standalone
   - Use the default link when the link is near a body of text or in appears in a body of text
   - Use the standalone link when there is separation from a body of text
+- When using multiple inline links together, be sure that their behaviors are consistent. For instance, don't have one link go to a new page and another one in the group trigger an action
+- Link buttons are often useful for reducing the visual clutter of large groups of actions. For example, when there are many cards in a group using the link button style instead of a true button style
```

Comment:
do we need to add anything to describe that users are expected to use link-buttons as buttons and not links? 
* valid modifiers to use when using the anchor element:
** default inline traditional navigation links
** standalone traditional link outside of copy block

WHEN LAUNCHING AN ACTION:  use the button rather than anchor element

* valid modifiers for when using the button element
** link-button: "spacing like a button, style like a link, action like a button
** default inline traditional  links
** standalone traditional link outside of copy block

Reply:
I think that's useful information, but also more dev-focused, and this section is more for designers. Is there somewhere else this is appropriate to include?

Reply:
I'm not sure I have always felt that the usage guidelines were misappropriated to the "design" side of things as the API really only outlines the options and not the usage guidelines - same gripe with the overview section being simplistic and not overly useful to any dev looking for a good "overview" of how to work with the component... it seems like less sections to get all the points covered would work better 
- but that could be future work I guess

Reply:
Yea, I guess we did remove the word 'design' so it is just under the 'guidelines' section for now - I was thinking about how developers would skip past a 'design guidelines' section. 

I tend to agree we can streamline our docs, and also am eager to get into that work. Especially after looking at the templates again and so many sections we don't utilize, knowing that people are scanning for visual examples, and also knowing that devs and designers reference the site differently. There's a lot we can do to improve/clarify I think.